scripts/scil_screenshot_bundle.py

```diff
@@ -0,0 +1,195 @@
+                                 Space.RASMM)
+
+    return new_sft, transformed_reference
+
+
+def main():
+    parser = _build_args_parser()
+    args = parser.parse_args()
+    required = [args.in_bundle, args.in_anat, args.target_template]
+    assert_inputs_exist(parser, required)
+
+    output_filenames = []
+    for axis_name in ['sagittal', 'coronal', 'axial']:
+        if args.output_suffix:
+            output_filenames.append(os.path.join(args.output_dir,
+                                                 '{0}_{1}.png'.format(
+                                                     axis_name,
+                                                     args.output_suffix)))
+        else:
+            output_filenames.append(os.path.join(args.output_dir,
+                                                 '{0}.png'.format(axis_name)))
+
+    assert_outputs_exist(parser, args, output_filenames)
+
+    if args.output_dir and not os.path.isdir(args.output_dir):
+        os.mkdir(args.output_dir)
+
+        if args.anat_opacity < 0.0 or args.anat_opacity > 1.0:
+            parser.error('Opacity must be between 0 and 1')
+
+    if args.uniform_coloring:
+        for val in args.uniform_coloring:
+            if val < 0 or val > 255:
+                parser.error('{0} is not a valid RGB value'.format(val))
+
+    # Get the relevant slices from the template
+    target_template_img = nib.load(args.target_template)
+
+    x_slice = int(target_template_img.shape[0] / 2)
+    y_slice = int(target_template_img.shape[1] / 2)
+    z_slice = int(target_template_img.shape[2] / 2)
+    slices_choice = (x_slice, y_slice, z_slice)
+
+    subject_data = prepare_data_for_actors(args.in_bundle, args.in_anat,
+                                           args.target_template)
+
+    # Create actors from each dataset for Dipy
+    sft, reference_data = subject_data
+    streamlines = sft.streamlines
+
+    volume_actor = actor.slicer(reference_data,
+                                affine=nib.load(args.target_template).affine,
+                                opacity=args.anat_opacity,
+                                interpolation='nearest')
+    if args.local_coloring:
+        colors = []
+        for i in streamlines:
+            local_color = np.gradient(i, axis=0)
+            local_color = np.abs(local_color)
+            local_color = (local_color.T / np.max(local_color, axis=1)).T
+            colors.append(local_color)
+    elif args.uniform_coloring:
+        colors = (args.uniform_coloring[0] / 255.0,
+                  args.uniform_coloring[1] / 255.0,
+                  args.uniform_coloring[2] / 255.0)
+    elif args.reference_coloring:
+        sft.to_vox()
+        streamlines_vox = sft.get_streamlines_copy()
+        sft.to_rasmm()
+        colors = []
+        normalized_data = reference_data / np.max(reference_data)
+        cmap = plt.get_cmap(args.reference_coloring)
+        for points in streamlines_vox:
```

Comment:
This could be replaced by an apply_along_axis since streamlines_vox is an ArraySequence

Reply:
Not sure how to call it, I can't make it work.
Will ask for help IRL

Reply:
I don't know how to call it, can't make it work.
Will ask for help IRL (if you know how to use it)